Refresher/UI/EmulatorPatchForm.cs

```diff
@@ -27,6 +27,7 @@ public EmulatorPatchForm() : base("RPCS3 Patch")
             AddField("RPCS3 dev_hdd0 folder", out this._folderField),
             AddField("Game to patch", out this._gameDropdown),
             AddField("Server URL", out this.UrlField),
+            AddField("Patch Digest", out this.PatchDigest),
```

Comment:
This checkbox shouldn't be here imho, should be autodiscover-only or under some form of advanced settings.

Also, this only gets added for the Emulator patch form, and might not be initialized elsewhere.

Reply:
Left as just a autodiscover only thing for now, advanced option can be added latter